backend/tests.py

```diff
@@ -82,4 +154,17 @@ def testSubmitGuess(self):
 
 
 if __name__ == "__main__":
-    unittest.main()
+    tester = TestBackend()
+    tester.setUp()
```

Comment:
`setup()` and `tearDown()` are run before and after each individual test. If you run `unittest.main()` it runs all test normally and you can specify tests in the command line.

Reply:
worked, just had to remove the `to=LobbyId`